.github/workflows/release.yml

```diff
@@ -23,9 +23,9 @@ jobs:
         with:
           distribution: temurin
           java-version: 17
-          server-id: ossrh
-          server-username: OSSRH_USERNAME
-          server-password: OSSRH_PASSWORD
+          server-id: central
```

Comment:
https://github.com/avides/avides-github-action-configuration/pull/19 - Auto-Config is already updated